eng/build.yml

```diff
@@ -17,36 +11,44 @@ phases:
     queue: ${{ parameters.queue }}
```

Comment:
change "publicBuildReasons" in the section above to "PullRequest".  This property is what the Arcade templates use to determine when to permit things like "telemetry" to be enabled for internal builds vs public builds.

If you haven't defined these variables, and "enableTelemetry" is "true", then you'll likely get a null reference exception when you try to queue (it should report the offending lines).

https://github.com/dotnet/arcade/blob/master/eng/common/templates/phases/base.yml#L22-L26

Arcade defines these variables here - https://github.com/dotnet/arcade/blob/master/eng/common/templates/phases/base.yml#L22-L26

Reply:
I have made the change, but I still can't queue up an official build.

Reply:
I don't see the changes for supporting telemetry.